pikaur/install_cli.py

```diff
@@ -34,7 +34,9 @@
 
 
 def get_editor():
-    editor = os.environ.get('EDITOR')
+    editor = os.environ.get('VISUAL')
```

Comment:
i'd prefer `editor = os.environ.get('VISUAL') or os.environ.get('EDITOR')` to reduce branching

the rest :+1: